markdown/ch7.markdown

```diff
@@ -95,11 +96,32 @@ whether jiggling it in one direction or another reduces their
 dissimilarity.  By doing this repeatedly, we can try to find the
 correct alignment.
 
+You'll remember our astronaut - Eileen Collins - from chapter 3.
```

Comment:
I believe my grammar is correct here, unless the USA style guide is substantially different from the UK/Australia one. Go check the rules, if you really think this should be commas.